examples/vision/reptile.py

```diff
@@ -0,0 +1,248 @@
+"""
+Title: Few-Shot learning with Reptile
+Author: [ADMoreau](https://github.com/ADMoreau)
+Date created: 2020/05/21
+Last modified: 2020/05/29
+Description: Reptile meta learning algorithm implemented in Keras
+
+"""
+
+"""
+The Reptile algorithm (https://arxiv.org/abs/1803.02999) was developed by OpenAI to
+perform model agnostic meta-learning. Specifically, this algorithm was designed to
+quickly learn to perform new tasks with minimal training, a task known as few-shot
+learning. The algorithm works by taking metagradients from the training of mini-batches
+representing new tasks, and using these gradients for a meta training step.
+"""
```

Comment:
Please add a section about the dataset: what the data is, what the goal is, why it's interested to use Reptile on this dataset.